CellTypeScan.md

```diff
@@ -0,0 +1,87 @@
+
+### Methodology
+This is the approach we have taken in building the infrastructure for this project:
+
+* Write Bash wrapper scripts to pass arguments to wrapped functions and retrieve output
+* Package scripts using Bioconda
+* Write workflows to execute tools in two scenarios:
+    * Tool evaluation where the ‘correct answer’ is known
+    * Cell type prediction for unannotated cell types
+
+![](img/CellTypeScan_methodology.png)
+
+
+### Performance evaluation 
+For testing cell typing methods against annotated cell types, tools will be evaluated using 5-fold cross-validation on training data set. A [nextflow pipeline]() has initially been developed to automate this task, followed by creating corresponding Galaxy components (see examples below). 
+
+![](img/label_analysis_galaxy.png)
+
+![](img/scpred_wf_galaxy.png)
+
+The statistics we use in assessment build on those used in [this study](https://github.com/tabdelaal/scRNAseq_Benchmark):
+* percentage of unlabelled cells
+* change in the percentage of unlabelled cells in predicted labels vs reference labels
+* proportion of exact matches between predicted and reference labels (exact matching)
+* average proportion of shared words between reference and predicted labels (partial matching)
+* accuracy & median F1-score
+* average semantic similarity between predicted and reference labels
+
+For cross-validation, final reported performance will be the mean of the performance achieved for each ‘fold’.
+
+### Integration with Single-cell Expression Atlas
+In a production scenario we are interested in generating reliable predictions for unseen data. A common strategy in this case is to use consensus-based predictions from multiple tools. Because each prediction tool requires its own index structure/ classifier, we will build a library of such classifiers that are then available for cell tying with new data. 
+
+Predictions will be collated and analysed for consistency using a two-step strategy. Firstly, predictions will be filtered at the level of individual tools. For each tool, classifier outputs that produce best-matching predictions will be selected based on similarity score (e.g. a distance metric or a p-value). Secondly, top predictions from each tool will be collated and analysed for consistency and semantic similarity. Tables with most likely consensus predictions and their corresponding scores will be produced. Similar to the performance evaluation use case, Nextflow pipelines were initially produced, replicated by Galaxy pipelines to provide garphical user interface. 
+
+
+### Results to date 
+#### Command-line interfaces
```

Comment:
4 hashes in a row don't seem to be rendered by markdown, maybe reduce hashing one level on all sections?